qt/python/mantidqtinterfaces/test/Muon/grouping_tab/pairing_table_group_selector_test.py

```diff
@@ -15,6 +15,7 @@
 from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_widget_presenter import PairingTablePresenter
 from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_widget_view import PairingTableView
 from mantidqtinterfaces.Muon.GUI.Common.test_helpers.context_setup import setup_context_for_tests
+from mantidqtinterfaces.Muon.GUI.Common.pairing_table_widget.pairing_table_constants import get_index_of_text
```

Comment:
Although I have opened an issue to investigate the reuse of get_index_of_text accross muon (#38315 ). The `get_index_of_text` function hsa been declared as a static method on the elemental analysis table view class, so you can access directly without creating the table object. 
```suggestion
from mantidqtinterfaces.Muon.GUI.ElementalAnalysis2.grouping_widget.ea_grouping_table_widget_view import EAGroupingTableView
```